includes/documents/class-wcpdf-invoice.php

```diff
@@ -585,6 +585,63 @@ public function get_ubl_settings_fields( $option_name ) {
 		return apply_filters( "wpo_wcpdf_{$this->type}_ubl_settings_fields", $settings_fields, $option_name, $this );
 	}
 
+	public function get_settings_categories(): array {
+		$settings_categories = array(
+			'init'             => array(
+				'title'   => '',
+				'members' => array( 'enabled' ),
+			),
+			'general'          => array(
+				'title'   => __( 'General', 'woocommerce-pdf-invoices-packing-slips' ),
+				'members' => array(
+					'attach_to_email_ids',
+					'disable_for_statuses',
+					'number_format',
+					'my_account_buttons',
+				),
+			),
+			'document_display' => array(
+				'title'   => __( 'Document display options', 'woocommerce-pdf-invoices-packing-slips' ),
+				'members' => array(
+					'display_email',
+					'display_phone',
+					'display_customer_notes',
+					'display_shipping_address',
+					'display_date',
+					'display_number',
+					'next_invoice_number' // this should follow 'display_number'
+				)
+			),
+			'due_date'         => array(
+				'title'   => __( 'Due date', 'woocommerce-pdf-invoices-packing-slips' ),
+				'members' => array(
+					'due_date',
+				)
+			),
+			'admin_display'    => array(
+				'title'   => __( 'Admin display options', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
Perhaps we can abbreviate this to just 'Admin', as there might be other options in the future that might fall into this section but are not strictly 'Admin display options'.